src/webhook_dispatcher.py

```diff
@@ -0,0 +1,16 @@
+import requests
+
+def post_to_webhook(webhook_url, job_id, filename, url, success=False):
+    payload = {
+        "source": "waas",
+        "job_id": job_id,
+        "success": success
+    }
+    if success:
```

Comment:
One could argue that we should also send the raw `result` JSON output to the webhook so the consumer could do whatever they wanted with the data without doing a file download

However the post body may be a bit big if the transcription is quite large

Reply:
I would argue that a webhook should be a notification of some state change, and not a a dump of all data. This makes it easier to maintain and the receiver can decide when, in which format and how they want to fetch the data. 
And especially in this case where the payload result might be a big bunch of data.

Reply:
Agree with @anderser